docs/tutorials/uploading-photo-to-a-product.md

```diff
@@ -0,0 +1,100 @@
+# Uploading Images to the OFF API
+<!--Add a brief introduction of what the tutorial does -->
+
+This basic tutorial shows you can upload image of a product to the OFF API.
+
+## Points to consider before uploading photos
+
+### Image Quality
+
+Uploading quality photos of a product, its ingredients and nutrition table is very important, since it allows the Open Food Facts OCR system to retrieve important data to analyze the product. The minimal allowed size for photos is 640 x 160 px.
+
+### Upload Behavior
+
+In case you upload more than one photo of the front, the ingredients and the nutrition facts, beware that only the first photo of each category will be displayed. (You might want to take additional images of labels, recycling instructions, and so on). All photos will be saved.
+
+### Label Languages
+
+Multilingual products have several photos based on languages present on the packaging. You can specify the language by adding a lang code suffix to the request.
+
+## Authentication
+
+The WRITE operations in the OFF API require authentication, therefore you need a valid `user_id` and `password`  to write the photo to 100% Real Orange Juice.
+
+> Sign up on the [Open Food Facts App](https://world.openfoodfacts.net/), to get your `user_id` and `password` if you dont have.
+For more details, visit the [Open Food Facts Authentication](https://openfoodfacts.github.io/openfoodfacts-server/introduction/api/#authentication).
+
+## Parameters
+
+### Code
+
+The barcode of the product.
+
+### Imagefield
+
+Imagefield indicates the type of the image you are trying to upload for a particular product. It can be either of these: `front`, `ingredients`, `nutrition`, `packaging` or `other`.
```

Comment:
```suggestion
`imagefield` indicates the type of the image you are trying to upload for a particular product. It can be either of these: `front`, `ingredients`, `nutrition`, `packaging` or `other`.
```